runtime/src/snapshot_utils/snapshot_storage_rebuilder.rs

```diff
@@ -418,6 +418,7 @@ impl SnapshotStorageRebuilder {
     /// Builds thread pool to rebuild with
     fn build_thread_pool(&self) -> ThreadPool {
         ThreadPoolBuilder::default()
+            .thread_name(|i| format!("solRbuildSnap{i:02}"))
```

Comment:
If we squeeze the threadpools down, we'll be able to get that e into Rebuild

Reply:
Potentially yeah.

These thread are only around during startup, and they are used to rebuild an accountsDb. The system doesn't have much else going on at the time, so we might actually be ok with this pool using so many threads (of course assuming that the work it is doing is more efficient with the higher number of parallelism)